articles/architecture-scenarios/_includes/_authorization/_introduction.md

```diff
@@ -1,34 +1,38 @@
-It’s important to start by distinguishing between Authentication, Authorization, and Access Control. Your Auth0 tenant (your Authorization Server) is typically responsible for Authentication and some or all of Authorization. Access Control however must be the responsibility of the Application or API itself, because access control is almost always contextual: 
+It’s important to start by distinguishing between Authentication, Authorization, and Access Control. Your Auth0 tenant (your Authorization Server) is typically responsible for Authentication and some or all of Authorization and Access Control. Additionally, ab Application or API itself may need to share Access Control responsibility, especially where contextual access is required: 
 
 * **Authentication**: the process of determining if a principal - a user or application - is who/what they say they are.
-* **Authorization**: the proceess of determining what is allowed, based on the principal, the consent they provide, what permissions they have been given, and/or the set of contextually specific access criteria. Broadly speaking, authorization in Auth0 breaks down into the following categories:
-  * **Consent**: what can be done on the behalf of a user principal (subject) or, in the case of an application principal authenticating via [Client Credentials](/flows/concepts/client-credentials) grant, what an administrative authority has authorized an application to do.
-  * **Access Control**: the access that is permitted. In general this can be thought of as breaking down into three sub-categories of granularity: 
+* **Authorization**: the proceess of determining what is allowed, based on the principal, the consent they provide, what permissions they have been given, and/or the set of contextually specific access criteria.
+* **Consent**: what can be done on the behalf of a user principal (subject) or, in the case of an application principal authenticating via [Client Credentials](/flows/concepts/client-credentials) grant, what an administrative authority has authorized an application to do.
+* **Access Control**: the access that is permitted. In general we typically group different types of access control into three distinct categories, so that it's easier to understand (a) which actor is responsible for storing the information, (b) which actor is responsible for making decisions, and (c) which is responsible for enforcing the restrictions. 
```

Comment:
```suggestion
* **Access Control**: Access control is really the umbrella concept that puts Authentication, Authorization, and Enforcement into one bucket.  In general we typically group different types of access control into three distinct categories, so that it's easier to understand (a) which actor is responsible for storing the information, (b) which actor is responsible for making decisions, and (c) which is responsible for enforcing the restrictions. 
```